scripts/shell_provisioner.sh

```diff
@@ -53,7 +53,7 @@ elif [ "$OSTYPE" = "Ubuntu" ]; then
         rm -f /tmp/puppetlabs.deb
         apt-get update
     fi
-elif [ "$OSTYPE" = "RedHat" ]; then
+elif [ "$OSTYPE" = "RedHat" ]  || [ "$OSTYPE" = "CentOS" ]; then
```

Comment:
CentOS sets the OSTYPE to RedHat too, where is that change coming from? I just want to be sure about it.